templates/posts/new.html.tera

```diff
@@ -27,14 +27,24 @@
         {% endfor %}
     {% endif %}
 
-     <label for="plume-editor">{{ "Content" | _ }}<small>{{ "Markdown is supported" | _ }}</small></label>
+    <label for="plume-editor">{{ "Content" | _ }}<small>{{ "Markdown is supported" | _ }}</small></label>
     <textarea id="plume-editor" name="content" rows="20">{{ form.content | default(value="") }}</textarea>
 
     {{ macros::input(name="tags", label="Tags, separated by commas", errors=errors, form=form, optional=true) }}
 
     {% set license_infos = "Default license will be {{ instance.default_license }}" | _(instance=instance) %}
     {{ macros::input(name="license", label="License", errors=errors, form=form, optional=true, details=license_infos) }}
 
+    <label for="cover">{{ "Illustration" | _ }}<small>{{ "Optional" | _ }}</small></label>
+    <select id="cover" name="cover">
+        <option value="none" {% if not form.cover %}selected{% endif %}>{{ "None" | _ }}</option>
+        {% for media in medias %}
+            <option value="{{ media.id }}" {% if form.cover == media.id %}selected{% endif %}>
```

Comment:
This is the exact line that is failing. I think you can't compare `form.cover` because `form` is `null` when creating a new article. `if not form.cover` (2 lines above) is legal as it verify the existence of the field, and its "nullness", but a compare only make sense if the field exist, it being null or not